src/test/scala/mesosphere/marathon/api/v2/json/V2AppDefinitionTest.scala

```diff
@@ -49,11 +49,13 @@ class V2AppDefinitionTest extends MarathonSpec with Matchers {
     shouldViolate(app, "id", idError)
 
     app = app.copy(id = "-dash-disallowed-at-start".toRootPath)
-    validateJsonSchema(app, false)
+    // The JSON schema does allow this. It is not as strict as our internal validation to keep the regular
+    // expression reasonably simple.
```

Comment:
At least, we are using the regular expression somewhere else then. Thank you for the pointer!

BTW, does that mean we have group support in the CLI? Any chance that will be available for universe packages soon?